Exercises/try-catch/try-catch.js

```diff
@@ -0,0 +1,25 @@
+//1. Make a function which will throw error with chance 50%.
+
+function error50() {
+  const random = Math.random();
+  if (random > 0.5) { throw new Error('Error. It\'s a pity'); }
+  console.log('No error');
+}
+
+//2. Make function which will execute function above, handle error and increase counter of function execution.
+// Counter hould save the value on each new function execution.
+
+function exeFunc() {
```

Comment:
In this code counter increasing is not connected to function execution. You should return function which both increases counter and tries to execute `error50` function